tensorflow_datasets/core/download/download_manager_test.py

```diff
@@ -112,6 +128,7 @@ def _extract(path, method, dest):
         extractor._Extractor, 'extract', side_effect=_extract).start()
 
   def setUp(self):
+    """Sets up the DownloadManager for the tests."""
```

Comment:
Similarly for `__eq__`,... `setUp` is already documented in the parent class.

Reply:
Fixed.